source/backend/dal/Repositories/PropertyLeaseRepository.cs

```diff
@@ -38,6 +38,25 @@ public IEnumerable<PimsPropertyLease> GetAllByPropertyId(long propertyId)
             return this.Context.PimsPropertyLeases.AsNoTracking().Include(p => p.Property).Include(l => l.Lease).Where(p => p.PropertyId == propertyId);
         }
 
+        /// <summary>
+        /// Get the property lease for the specified lease id.
+        /// </summary>
+        /// <param name="leaseId"></param>
+        /// <returns></returns>
+        public IEnumerable<PimsPropertyLease> GetAllByLeaseId(long leaseId)
+        {
+            return this.Context.PimsPropertyLeases.AsNoTracking().Include(p => p.Property).Include(l => l.Lease).Where(p => p.LeaseId == leaseId);
+        }
+
+        /// <summary>
+        /// Delete a propertyLeaseFile. Note that this method will fail unless all dependencies are removed first.
```

Comment:
Seems like we could use some unit tests to cover this methods

Reply:
These methods don't actually fall into our minimal requirement for unit tests: https://confluence.th.gov.bc.ca/display/PIMS/Best+Practices#BestPractices-UnitTests

So nice to have for sure, but minimal value since we would mostly just be testing the underlying EF methods.